cmd/helm/history.go

```diff
@@ -104,9 +105,13 @@ func (r releaseHistory) WriteYAML(out io.Writer) error {
 
 func (r releaseHistory) WriteTable(out io.Writer) error {
 	tbl := uitable.New()
-	tbl.AddRow("REVISION", "UPDATED", "STATUS", "CHART", "APP VERSION", "DESCRIPTION")
+	tbl.AddRow("REVISION", "UPDATED", "STATUS", "CHART", "APP VERSION", "DESCRIPTION", "RELEASE TAG")
```

Comment:
According to the [HIP on backwards compatibility](https://github.com/helm/community/blob/main/hips/hip-0004.md), "the format of structured output (tables, JSON, YAML) must not change". Adding a column changes the structure of the output. We have scripts that parse the output. This means the display change needs to be behind a flag.

Helm search of the hub already has an example of this... https://github.com/helm/helm/blob/d9f776bb3561647378b9ff588bb5042fbd37987c/cmd/helm/search_hub.go#L143-L147

Reply:
No problem ill add that in.

Reply:
@mattfarina are thinking i should create a like searchhub does?
```
type hubSearchWriter struct {
	elements    []hubChartElement
	columnWidth uint
	listRepoURL bool
}
```
so it would be like this
```
type releaseHistoryWriter struct {
  elements         []releaseInfo
  listReleaseTag bool
}
```

Reply:
@mattfarina just following up for clarity ^

Reply:
@mattfarina Happy New Year!

Just wanted to follow up on this to see if i can get a little clarification.